index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;500;600;700&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>Document</title>
+</head>
+<body>
+     <section class="basic-section">
+      <header class="header">
+       <nav class="nav">
+        <div class="logo">
+            <h3>KMarketer</h3>
+        </div>
+            <ul class="nav_bar">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Product</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a><img alt="svg" src="img/Vector.png"></a></li>
+                <li><a><img alt="svg" src="img/icnshopping-cart.icn-xs.png"></a></li>
```

Comment:
Does it work with such long name? src="img/icnshopping-cart.icn-xs.png"> Maybe to rename?

Reply:
agree, better choose a more clear and short name. like `cart-icn`